configure.py

```diff
@@ -384,12 +384,16 @@ def set_action_env_var(environ_cp,
 def convert_version_to_int(version):
   """Convert a version number to a integer that can be used to compare.
 
+  Version strings of the form X.YZ and X.Y.Z-xxxxx are supported. The
```

Comment:
Should we check that the leading parts are not zero? Otherwise the version number will not be unique, e.g. 0.23 and 0.0.23 would results in the same integer.